Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM adoptopenjdk/openjdk11:alpine-jre
```

Comment:
As you may see, the project uses Java 8.

Reply:
downgraded :1st_place_medal: